addon-resizer/nanny/nanny_lib.go

```diff
@@ -87,26 +87,25 @@ func PollAPIServer(k8s KubernetesClient, est ResourceEstimator, contName string,
 			log.Error(err)
 			continue
 		}
-		log.Infof("The number of nodes is %d", num)
+		log.V(4).Infof("The number of nodes is %d", num)
 
 		// Query the apiserver for this pod's information.
 		resources, err := k8s.ContainerResources()
 		if err != nil {
 			log.Errorf("Error while querying apiserver for resources: %v", err)
 			continue
 		}
-		log.Infof("The container resources are %+v", *resources)
 
 		// Get the expected resource limits.
 		expResources := est.scaleWithNodes(num)
-		log.Infof("The expected resources are %+v", *expResources)
 
 		// If there's a difference, go ahead and set the new values.
 		if !shouldOverwriteResources(int64(threshold), resources.Limits, resources.Requests, expResources.Limits, expResources.Requests) {
-			log.Infof("Resources are within the expected limits.")
+			log.V(4).Infof("Resources are within the expected limits. Actual: %+v Expected: %+v", *resources, *expResources)
```

Comment:
Optional: As for now I can see resources in the following format
```
cpu:{i:{value:88 scale:-3} d:{Dec:<nil>} s:88m Format:DecimalSI}
```
How about making it more readable for a human reader?